.gitignore

```diff
@@ -1 +1,3 @@
 .DS_Store
+.idea
+.idea/*
```

Comment:
Please remove these, these PHPStorm specific files should be ignored from a global `.gitignore` file in your own home directory, see https://stackoverflow.com/a/7335487/1911294 for how to do that.

Reply:
:+1: